scss/inc/base/_list-style.scss

```diff
@@ -1,5 +1,10 @@
 ul, ol {
     padding-left: 40px;
+    padding-right: 3px;
```

Comment:
I had here the same issue as you on my ticket.
I solved it with 
`body[dir="rtl"]`

Reply:
changes above should fix the issue